<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParaDialog_stranno</class>
 <widget class="QDialog" name="ParaDialog_stranno">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atlas Guided C.elegans Cell Annotation</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>710</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_imgv3d">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Image from V3D:</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_imgv3d">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>251</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_atlasfile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Atlas from File:</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_atlasfile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atlasfile">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Image &amp;&amp; Atlas Import</string>
   </property>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_celloifile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Import interesting cell names from File:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_celloifile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>371</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_celloifile">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_celloifile_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_celloifile_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>371</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_celloi_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>74</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Import secondary interesting cell names from File:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>221</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Initial Align paras:</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_refchannel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_refchannel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>reference channel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_downsampleratio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>downsample ratio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temperature_ini">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>initial temperature:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temperature_min">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>minimal temperature:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_annealingrate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>annealing rate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_iter_pertemp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>iter per temperature:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_downsampleratio">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_temp_ini">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_temp_min">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_annealingrate">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_niter_pertemp">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>(R: 1, G: 2, B:3)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>241</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Refine Align paras:</string>
   </property>
   <widget class="QCheckBox" name="checkBox_refinealign_simple">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Simple refined align</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>stop iteration when:</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>iter num &gt; </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ref_maxiter">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ref_minposchange">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>total pos channge &lt; </string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ref_cellradius">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>fixed temperature:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ref_temperature">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>cell radius:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>(fast but less accurate)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output deformed Atlas to:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_atlas_output">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atlas_output">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_seglabel_output">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output seg-label image to:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_seglabel_output">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Visulization</string>
   </property>
   <widget class="QCheckBox" name="checkBox_showatlas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>show deformed atlas</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_showsegmentation">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>show segmentation</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fgthresh">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>560</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_fgthresh">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FG thresh factor:</string>
   </property>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>buttonBox</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>groupBox_8</zorder>
  <zorder>groupBox_9</zorder>
  <zorder>lineEdit_fgthresh</zorder>
  <zorder>label_fgthresh</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ParaDialog_stranno</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>700</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ParaDialog_stranno</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>700</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_imgv3d</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_imgv3d</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_refinealign_simple</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_8</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_refinealign_simple</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_ref_cellradius</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_celloi_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_celloifile_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_celloi_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_celloifile_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
